mozilla-release/browser/components/search/extensions/cliqz/_locales/de/messages.json

```diff
@@ -6,15 +6,15 @@
     "message": "Cliqz"
   },
   "searchUrl": {
-    "message": "https://suchen.cliqz.com/"
+    "message": "https://beta.cliqz.com/search?q={searchTerms}"
```

Comment:
@tsl143 I think it would make sense to add lang=de as a GET parameter to the German version.
So it could be
```
https://beta.cliqz.com/search?q={searchTerms}&lang=de
```

Reply:
This will be dealt by client side via `accept language` headers